raw-data/PH/wip_inventories.tsv

```diff
@@ -900,3 +900,136 @@ rose2021mojenotrinitario.pdf	trin1274	Mojeño	Trinitario	p		p
+				b		w	"It occurs only before /a/ when following /u/"	
+						β	"It occurs in all other positions"	
+				s		s		
+				ɕ		ʃ	"It occurs only word initial when followed by the central low vowel /a/"	
+						ɕ	"It occurs word initial before /i/ and word medial after /i/ and /e/ and after all nasalized vowels"	
+						ʂ	"It occurs in all other positions"	
+				m		m		
+				n		n		
+				ɲ	"/ɲ/ cannot occur before /i/"	ɲ		
+				ɾ		ɾ		
+				j		j		
+				i		ɪ	"It occurs in vowel clusters as the second vowel if the first vowel is of a low quality, (/e/, /a/, /o/)"	
+						i	"It occurs in all other positions"	
+				e		ɛ	"It occurs in vowel clusters as the second vowel if the first vowel is of a low quality, (/a/, /o/)"	
+						e	"It occurs in all other positions"	
+				a	"voiced low open central spread vowel"	a		
+				o		ɔ	"[ɔ] and [o] fluctuate, [ɔ] being more dominant"	
+						o	"[ɔ] and [o] fluctuate, [ɔ] being more dominant"	
+				u		ʊ	"[ʊ] and [u] fluctuate, but word initial the tendency is strongly toward [u] while word final the tendency is towards [ʊ], sometimes as far open as [o]"	
+						u	"[ʊ] and [u] fluctuate, but word initial the tendency is strongly toward [u] while word final the tendency is towards [ʊ], sometimes as far open as [o]"	
+				ɨ		ɨ		
+				ĩ		ɪ̃	"It occurs in vowel clusters as the second vowel if the first vowel is of a low quality, (/e/, /a/, /o/)"	
+						ĩ	"It occurs in all other positions"	
+				ẽ		ɛ̃	"It occurs in vowel clusters as the second vowel if the first vowel is of a low quality, (/a/, /o/)"	
+						ẽ	"It occurs in all other positions"	
+				ã		ã		
+				õ		ɔ̃	"[ɔ] and [o] fluctuate, [ɔ] being more dominant"	
+						õ	"[ɔ] and [o] fluctuate, [ɔ] being more dominant"	
+				ũ		ʊ̃	"[ʊ] and [u] fluctuate, but word initial the tendency is strongly toward [u] while word final the tendency is towards [ʊ], sometimes as far open as [o]"	
+						ũ	"[ʊ] and [u] fluctuate, but word initial the tendency is strongly toward [u] while word final the tendency is towards [ʊ], sometimes as far open as [o]"	
+				ɨ̃		ɨ̃		
+				iː	"Complex nuclei consisting of long vowels always occur in stressed positon"	iː		
+				aː	"Complex nuclei consisting of long vowels always occur in stressed positon"	aː		
+				oː	"Complex nuclei consisting of long vowels always occur in stressed positon"	oː		
+				uː	"Complex nuclei consisting of long vowels always occur in stressed positon"	uː		
+				ɨː	"Complex nuclei consisting of long vowels always occur in stressed positon"	ɨː		
+				ĩː	"Complex nuclei consisting of long vowels always occur in stressed positon"	ĩː		
+				ãː	"Complex nuclei consisting of long vowels always occur in stressed positon"	ãː		
+				õː	"Complex nuclei consisting of long vowels always occur in stressed positon"	õː		
+				ũː	"Complex nuclei consisting of long vowels always occur in stressed positon"	ũː		
+				ɨ̃ː	"Complex nuclei consisting of long vowels always occur in stressed positon"	ɨ̃ː		
+				ia		ia		
```

Comment:
The PDF states "Vowel sequences can either function as a complex nucleus, or a sequence of syllables with the patterns CV.V or V.V. The conditioning factor is stress."
A near minimal pair is /na.'i.bi/ 'your dress' and /na.nai.'ña/ 'all'.
The way diphthongization is phrased makes it seem like stress on the second vowel and diphthongization are in complimentary distribution, where a VV sequence is V.V iff stress falls on the second vowel and VV otherwise. Unfortunately the analysis does not state whether stress is contrastive, but since they mention that the "conditioning factor is stress" I think you're right, they're probably non-contrastive. Thoughts before I update the PR?

Reply:
@bambooforest I wanted to check in on this before I update it, let me know what you think, thanks!

Reply:
My thoughts remain the same -- maybe you can have a quick look @drammock ?

Reply:
Fine with me. Thx!